go.mod

```diff
@@ -169,6 +168,7 @@ require (
 
 // Below are the long-lived replace of the Cosmos SDK
 replace (
+	cosmossdk.io/x/params => ./x/params
```

Comment:
Let's not introduce params as an SDK dependency.
We should complete https://github.com/cosmos/cosmos-sdk/issues/17360 first.

Reply:
Why is this needed?